README.md

```diff
@@ -34,13 +36,13 @@ python3 -m venv venv && source venv/bin/activate
 pip install -r requirements.txt && npm install && npm run build
 ```
 
-4. Build all targets at once:
+4. Build your desired platform target.
 
 ```
-make all
+make linux
 ```
 
```

Comment:
Could add multi-builds. The output gets a bit hard to parse vs serially building, but

```
make linux k8s container
```

Is my personal standard batch when adding new tutorials.

Reply:
This started working for me, unclear why but seems solid now. Adding multiple target example.